src/Data/Textile/Simulator.elm

```diff
@@ -437,9 +446,7 @@ computeFabricImpacts db ({ inputs, lifeCycle } as simulator) =
                                     , yarnSize =
                                         inputs.yarnSize
                                             |> Maybe.withDefault
-                                                (inputs.product.yarnSize
-                                                    |> Maybe.withDefault Unit.minYarnSize
-                                                )
+                                                inputs.product.yarnSize
```

Comment:
Nit: for legibility, this can fit on its own line `|> Maybe.withDefault inputs.product.yarnSize`